torch/_inductor/config.py

```diff
@@ -318,6 +318,29 @@ def is_fbcode():
 # the default to spawn.
 worker_start_method = "fork"
 
+# Flags to turn on all_reduce fusion. These 2 flags should be automaticaly turned
+# on by DDP and should not be set by the users.
+fuse_ddp_communication = False
+fuse_ddp_bucket_size = 25
```

Comment:
curious: if they are not meant to be set by users, should we prepend the name with `_` to indicate it's private?

Reply:
wonder do we still plan to do this?

Reply:
oh, yes, I modified this locally. The landed version will have no public configs.